mentoring/programs/meet-our-contributors.md

```diff
@@ -8,80 +17,85 @@ about our upstream community, watch interviews with our contributors, and
 participate in peer code reviews.
 
 ## When:
-Every first Wednesday of the month at the following times. Grab a copy of the
-calendar to yours from [kubernetes.io/community]
-* 03:30pm UTC [Convert to your timezone].
 
-## Where:  
-Follow along live:  
-[Kubernetes YouTube Channel]
+Every first Wednesday of the month at the following times:
+- 03:30pm UTC [Convert to your timezone].
+
+## Where:
+
+Follow along live: [Kubernetes YouTube Channel]
+
+Ask questions: [#meet-our-contributors] slack channel
```

Comment:
Inconsistent casing of "slack". it's "Slack" elsewhere?

Reply:
fixed 👍